src/plugins/flame-chart-plugin.ts

```diff
@@ -174,8 +174,8 @@ export default class FlameChartPlugin extends UIPlugin {
         this.initialClusterizedFlatTree = clusterizeFlatTree(
             this.metaClusterizedFlatTree,
             this.renderEngine.zoom,
-            this.min,
-            this.max
+            this.min ?? 0,
```

Comment:
I think it's preferable to use the null check in the `clusterizeFlatTree` function.

Reply:
sorry, didn't make time yesterday to fix this
will push the fix today during the day